### PR TITLE
Edit an exercise entry to implement

### PR DESCRIPTION
**Pull request by LWachtel1**
Implements #10 
Implements editExercise() in ExerciseCommand class and corresponding editExercise() in ExerciseManager class
Adds corresponding tests in ExerciseCommandTest class

### CODE REVIEW

File: src/main/java/byteceps/commands/ExerciseCommand.java [hunk 1]
```diff
@@ -65,4 +67,39 @@ public CommandResult listExercises() {
...
+    public CommandResult editExercise() {
+        String prevExerciseName = getActionParameters().trim();
+        ArrayList<InputArguments> newExercise = getAdditionalArguments();
+
+
+        if (prevExerciseName == null || prevExerciseName.isEmpty()) {
+            return new CommandResult("Previous exercise name cannot be empty.");
+        }
+
+        if (newExercise.size()!=1 ) {
```
Comment (joshualeejunyi) [inline src/main/java/byteceps/commands/ExerciseCommand.java:80]: I think the formatting is not standard?
```suggestion
        if (newExercise.size() != 1) {
```

File: src/main/java/byteceps/exercises/ExerciseManager.java [hunk 2]
```diff
@@ -47,4 +47,17 @@ public boolean hasExercise(String exerciseName) {
         }
         return false;
     }
+
+    public void editExercise(String prevExerciseName, String newExerciseName) {
+        Exercise newExercise = new Exercise(newExerciseName);
+
+        for (Exercise exercise : exercises) {
+
```
Comment (joshualeejunyi) [inline src/main/java/byteceps/exercises/ExerciseManager.java:55]: Remove the whitespace

**Review by joshualeejunyi — CHANGES_REQUESTED** [review]

File: src/main/java/byteceps/commands/ExerciseCommand.java (hunk 1 above)
Comment (LWachtel1) [reply src/main/java/byteceps/commands/ExerciseCommand.java]: Done

File: src/main/java/byteceps/exercises/ExerciseManager.java (hunk 2 above)
Comment (LWachtel1) [reply src/main/java/byteceps/exercises/ExerciseManager.java]: Done

File: src/main/java/byteceps/commands/ExerciseCommand.java [hunk 3]
```diff
@@ -65,4 +67,39 @@ public CommandResult listExercises() {
             return new CommandResult(exerciseList.toString());
         }
     }
+
+    public CommandResult editExercise() {
+        String prevExerciseName = getActionParameters().trim();
```
[the remark this answers is not in the document]
Comment (LWachtel1) [reply src/main/java/byteceps/commands/ExerciseCommand.java:72]: trim() seems to be required to avoid bug (see telegram GC for photos)

**Pull request merged**